types/SocketEvent.js

```diff
@@ -13,6 +13,7 @@ export default {
   SET_IS_RANKED: 'setIsRanked',
   REJECT_STALEMATE: 'stalemateReject',
   REQUEST_STALEMATE: 'stalemateRequest',
+  ACCEPT_STALEMATE: 'stalemateAccept',
```

Comment:
Should probably rename these to match move type:

STALEMATE_REQUEST, STALEMATE_ACCEPT, STALEMATE_REJECT